mappings/net/minecraft/entity/passive/AllayEntity.mapping

```diff
@@ -0,0 +1,11 @@
+CLASS net/minecraft/unmapped/C_ueeoxhql net/minecraft/entity/passive/AllayEntity
+	FIELD f_ecxyagii ITEM_PICKUP_REACH Lnet/minecraft/unmapped/C_ceivtqhh;
+	FIELD f_flhfrvyz inventory Lnet/minecraft/unmapped/C_rsloiwzx;
+	FIELD f_hexlvlut entityPositionSource Lnet/minecraft/unmapped/C_jwdpbxxx;
+	FIELD f_hnqqqvvl GAME_EVENT_LISTENER_RANGE I
+	FIELD f_iqgfejuv MEMORY_MODULES Lcom/google/common/collect/ImmutableList;
+	FIELD f_owidlbwe dynamicGameEventListener Lnet/minecraft/unmapped/C_wpaexsvv;
+	FIELD f_qsyixpba USE_V2_MOVE_PARTICLES Z
+	FIELD f_xaexqzwn SENSORS Lcom/google/common/collect/ImmutableList;
+	METHOD m_amirmypp createAttributes ()Lnet/minecraft/unmapped/C_sdjuuzrz$C_tehwrjus;
```

Comment:
```suggestion
	METHOD m_amirmypp createAllayAttributes ()Lnet/minecraft/unmapped/C_sdjuuzrz$C_tehwrjus;
```

Reply:
Isn't that pointless when the class is AllayEntity anyway?

Reply:
I think all the attributes methods use that naming scheme

Reply:
It makes more sense to not have the Allay added in IMO, It's also how the whole of #120 is done

Reply:
Agree with oro, make it createAllayAttributes. If you look at Horse they do the same thing, its kinda standard from what im aware of.